cmd/titus-reaper/main.go

```diff
@@ -18,9 +18,10 @@ func main() {
 	flag.BoolVar(&debug, "debug", false, "Turn on debug logging")
 	flag.Parse()
 
-	logsutil.MaybeSetupLoggerIfUnderSystemd()
 	if debug {
 		log.SetLevel(log.DebugLevel)
+	} else {
+		logsutil.MaybeSetupLoggerIfOnJournaldAvailable()
```

Comment:
Why do this only when !debug?

Reply:
Because if you're running with -debug, you're probably running from the console, and not a service?